packages/documentation/docs/controls/_blind_styleguide.md

```diff
@@ -10,76 +10,58 @@ import SourceAngularHtml from './../auto-generated/previews/angular/blind.html.m
 
 import Playground from '@site/src/components/Playground'
 
-Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element.
+Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element. Blinds can display a large amount of content in a compact space or present information in an organized and hierarchical way. Blinds reduce the user's cognitive load by removing clutter and less important information from an interface. We typically don't use blinds if the content is central to the user's task due to its reduced visibility and accessibility.
 
-![Blind overview](/img/pattern_illustrations/Blind_overview.png)
+Blinds consist of a header section on the top and a content section below. The header section contains a chevron on the left followed by the blind's label. Within the content section, content can be placed freely.
 
-Go to:
+![Blind overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2%3A2&mode=design&t=9faEnH99BaAxqCGM-1)
 
-- [States](#states)
-- [Dos and don't](#dos-and-donts)
+1. Header section
+2. Content section
 
-## Usage
+## Types
 
-Blinds are often used to display a large amount of content in a compact space, or to present information in an organized and hierarchical way.​ Blinds can reduce cognitive load of the user be removing clutter and less important information from an interface.
+Multiple blinds can be placed below each other to create an accordion. The recommended distance between the blinds is `0.5rem`. Blinds within an accordion can either close and open independently or dependently depending on the design requirements.
```

Comment:
```suggestion
Multiple blinds can be placed below each other to create an accordion. The recommended distance between the blinds is `0.5rem`. The respective design requirements dictate whether it is allowed have multiple open blinds or if there can only be one blind open at a time.
```

Reply:
I tried to fix the description to make it more clear and straightforward. Unfortunately, I couldn't reach Jennie for a review.